app/views/layouts/blacklight/base.html.erb

```diff
@@ -41,5 +41,6 @@
     <noscript><%= render partial: 'shared/footer_noscript' %></noscript>
     <%= render partial: 'shared/footer' %>
     <%= render partial: 'shared/modal' %>
+    <%= content_for(:correction_modal) %>
```

Comment:
How does this work? Does it not have to be `yield`ed?

Interesting to place it with the layout but render it from the show pages as a `content_for` block. I'm just not sure how this bit works.

Reply:
It's in the layout because it needs to be in a specific spot in the DOM or else the modal doesn't work. I use `content_for` because I don't want the form to always be rendered.

Documentation for `content_for` is here: https://guides.rubyonrails.org/layouts_and_rendering.html#using-the-content-for-method

Reply:
Yes I'm with you and in general the approach makes sense, but the `content_for` is what you use to wrap the content. Doesn't it need to be yielded? like see above in line 36. I assume it works, though. maybe we could look at it together.

Reply:
Oh, yeah, `content_for` with a name and a block is equivalent to yielding to a symbol in the view. See https://github.com/pulibrary/pulfalight/pull/476#discussion_r521556966.

Reply:
> Oh, yeah, content_for with a name and a block is equivalent to yielding to a symbol in the view. See https://api.rubyonrails.org/classes/ActionView/Helpers/CaptureHelper.html#method-i-content_for.

Aha! you mean "with a name and without a block" I think.

So we could rewrite line 36 this way? (Not proposing to do that on this pr just hypothetically.)

Reply:
Yes